.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,108 @@
+name: Publish Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        type: boolean
+        description: "Prerelease"
+        # Whether to create a prerelease or proper release
+        default: true
+        required: true
+      swift_format_version:
+        type: string
+        default: 601.0.0
+        description: "swift-format version"
+        # The version of swift-format to tag. If this is a prerelease, `-prerelease-<date>` is added to this version.
+        required: true
+      swift_syntax_tag:
+        type: string
+        default: 601.0.0
+        description: "swift-syntax version"
+        # The swift-syntax version to depend on. If this is a prerelease, the latest swift-syntax prerelease tag for this version is used.
+        required: true
+
+jobs:
+  check_triggering_actor:
+    name: Check user is allowed to create release
+    # Only a single user should be allowed to create releases to avoid two people triggering the creation of a release
+    # at the same time. If the release manager changes between users, update this condition.
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          if [[ "${{ github.triggering_actor }}" != "ahoppen" ]]; then
```

Comment:
We should create a team for swift-format, so it's not blocked on one person.

Reply:
My thought was that it’s pretty easy to modify the workflow to shift responsibilities. That way the responsibility transfer is explicit.